src/game.py

```diff
@@ -47,7 +47,27 @@ def create(self, screen, messages):
             self.image = self.font.render('NO IMAGE', False, (255, 255, 255))
 
         self.label = self.font.render(self.title, False, (255, 255, 255))
-        self.label_desc = self.font.render(self.description, False, (255, 255, 255))
+        self.desc_lines = self.wrapDesc(self.description, self.font)
+    
+    def wrapDesc(self, description, font):
+        lines = []
+        self.desc_split = description.split(" ")
+        self.curr_width = font.size(description)[0]
+        self.line2 = ""
```

Comment:
Ah, I didn't see the new PR. Don't forget to change this variable name and then I'll merge

Reply:
Sorry about that! It's now updated.